tests/test_translation_display.py

```
@@ -0,0 +1,73 @@
+import pytest
+from translate_srt import TranslationDisplay, MODEL_PRICING
+
+@pytest.fixture
+def display():
+    """Fixture to create a TranslationDisplay instance"""
+    return TranslationDisplay("gpt-4-1106-preview")
+
+def test_display_update_header(display):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_display_update_header`

Add return type annotation: `None`

(ANN201)